task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +115,17 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function TryFunc(){
+        for (; attempts > 0;){
```

Comment:
just use a normal for(let i =0; exitcondition; i++) loop

Reply:
Fixed

Reply:
not fixed. Increment should also be in the loop defenition. A general note 99% of the time you should keep increment in the loop definition. This way it's way easier to read and most of the time it's unnecessary to move it in different place.